packages/orchestration/src/examples/swapExample.contract.js

```diff
@@ -102,12 +101,6 @@ const contract = async (zcf, privateArgs, zone, { orchestrate, zoeTools }) => {
   const { brands } = zcf.getTerms();
 
   /** deprecated historical example */
-  /**
-   * @type {OfferHandler<
-   *   unknown,
-   *   { staked: Amount<'nat'>; validator: CosmosValidatorAddress }
-   * >}
-   */
```

Comment:
Should we have a flow-friendly `OfferHandler` type?

Reply:
Not sure what you have in mind. The deleted lines work just fine. I removed them because they're redundant now with the inferred type

Reply:
Ah I didn't realize offerArgs were typed on L24-L26